src/components/AppNavigation/EditCalendarModal/ShareItem.vue

```diff
@@ -153,6 +173,12 @@ export default {
 
 	&__label {
 		flex: 1 auto;
+		flex-direction: column;
+
+		p {
+			color: var(--color-text-lighter);
+			line-height: 1;
```

Comment:
that seems odd. does it miss a unit?

Reply:
No this is fine, it's based on a proportion to the font size, and the styling is copied from a similar component